docs/get-started/start-teku.md

```diff
@@ -125,6 +125,24 @@ Content-Length: 0
   </TabItem>
 </Tabs>
 
+## Exit Codes
+
+**Exit Code 1**
+
+This code indicates a scenario where Teku has exited with a fatal error, however restarting Teku without changes is a logical step in correcting it.
+
+Example: If the beacon chain controller has issues starting P2P services on a particular port, the system will exit with a fatal error code of 1. This could correct itself with a restart.
+
+**Exit Code 2**
+
+This code indicates a scenario where Teku has exited with a fatal error, but will not be corrected by itself if Teku is restarted.
```

Comment:
Probably worth explicitly saying that most user config errors fall into this category.